src/Scenes/GameScene/GameScene.ts

```diff
@@ -32,7 +33,7 @@ export class GameScene implements Scene {
 		this._layer.addChild(this.sessionRenderer);
 		this._layer.addChild(this._viewManager);
 
-		this.sessionRenderer.sync(this._session.level);
+		this.sessionRenderer.sync(this._session.level, new TurnState(this._session.level));
```

Comment:
Do we need to pass in both the level and turn state? Turn state already contains a reference to the same level.